1-Draft/RFC00XX-Update-Error-View.md

```diff
@@ -0,0 +1,181 @@
+---
+RFC: Update $ErrorView with Single String Error Messages
+Author: Jason Helmick
+Status: Draft
+Version: 0.0.0
+Area: PowerShell
+Comments Due: 
+---
+
+# Update $ErrorView with Single String Error Messages
+
+When an error occurs in PowerShell, the customers on-screen error message experience currently
+provides a level of detail that obscures the inner exception message from being read by
+new or occasional PowerShell users.
+To improve both comprehension and the troubleshooting experience, a single string of text
+containing the error will be displayed.  A cmdlet will provide the full error details when desired.
+
+## Motivation
+
+The on-screen experience, when receiving an error message,
+is controlled through the views NormalView (the default) and CategoryView. These are user selectable though the preference variable $ErrorView.
+The addition of a “ConciseView” as default that contains only specific error
+information producing a single line on screen and in logs will improve customer success.
+
+Comprehensive error information is available to the customer through
+accessing the error object. For new and occasional customers,
+a cmdlet “Resolve-ErrorRecord” will simplify the process of getting detailed error information, improving customers success.
+
+## Specification
+
+The proposal is to add a new single-line default error message view called ConciseView
+and add a new cmdlet (Resolve-ErrorRecord) to provide detailed error information.
+
+__Key Design Considerations__
+
+1. To reduce confusion and improve debugging success,
+error messages by default should produce a single line of text, including the word “ERROR:” to make consistent with Verbose and Warning messages. This is added to $errorView as view named "ConciseView".
+
```

Comment:
Regarding the prefix text, some consideration for [issue #10236](https://github.com/PowerShell/PowerShell/issues/10236) and whether the prefix is added by the host or the F&O layer is necessary.

Reply:
Issue #10236 is not a small item and would be a big breaking change.  For now, we should clarify this line to say that [WriteErrorLine](https://docs.microsoft.com/en-us/dotnet/api/system.management.automation.host.pshostuserinterface.writeerrorline?view=pscore-6.2.0) will be called instead of implying different code would add the "ERROR:" prefix